test/functionalTest/cases/3045_wrong_inclusion_location_metadata_notification/wrong_inclusion_location_metadata_V2.test

```diff
@@ -0,0 +1,370 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This test is covering the case

![imagen](https://user-images.githubusercontent.com/1534240/48831407-0e344d80-ed77-11e8-8bb0-841db931e3fd.png)

Great! 

NTC

Reply:
It's ok